core/prisma/migrations/20241002220742_remove_label_column_from_inputs/migration.sql

```diff
@@ -0,0 +1,15 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `label` on the `form_elements` table. All the data in the column will be lost.
```

Comment:
hm is this true? I don't think the column `label` was dropped?

Reply:
oops yes i manually edited this and left some bad comments in, thanks!